add-on/panels/js/panel.jsx

```diff
@@ -340,7 +340,7 @@ loop.panel = (function(_, mozL10n) {
       if (this.props.userProfile && this.props.userProfile.email) {
         return (
           <div className="user-identity">
-            {loop.shared.utils.truncate(this.props.userProfile.email, 24)}
+            {loop.shared.utils.truncate(this.props.userProfile.email, 40)}
```

Comment:
I just happened to flick on to this PR... This landed in [bug 1183636](https://bugzilla.mozilla.org/show_bug.cgi?id=1183636), but I don't understand why we've got a special truncate function rather than using layout with `text-overflow: ellipsis;`.

@mikedeboer do you remember the reasoning behind this?